implementations.md

```diff
@@ -0,0 +1,19 @@
+# IPFS Implementations
+
+For a list of IPFS API Client Libraries, please refer to *[Client Libraries](https://github.org/ipfs/ipfs/client-libraries.md)*
+
+| Language | Project |
```

Comment:
maybe add a column for "completeness" so users can know what's good to use, what's being worked on, etc. either with a qualitative call, a presumed percent, or even a ratio based on API calls implemented.